### ISSUE TITLE
Only supported by recent C# language versions

### ISSUE THREAD

**Issue by hrumhurum** [dotnet/docs#5274]
It is worth noting what specific C# versions 'private protected' is supported by.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ee5e6972-0f7a-8bb7-9f5a-0ba75909f9d8
* Version Independent ID: 618c0361-38a0-4d6f-f46a-881db2e4b399
* Content: [private protected (C# Reference)](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/private-protected)
* Content Source: [docs/csharp/language-reference/keywords/private-protected.md](https://github.com/dotnet/docs/blob/master/docs/csharp/language-reference/keywords/private-protected.md)
* Service: **dotnet-csharp**
* GitHub Login: @sputier
* Microsoft Alias: **wiwagn**

**Comment by BillWagner** [comment]
`private protected` was added in C# 7.2

**Comment by vikas304** [comment]
'private protected ' is giving an error when we are using both simultaneously.

**Comment by rpetrusha** [comment]
@vikas304, `private protected` is available starting with C# 7.2. To use it, you have to have at least C# 7.2 installed, and you have to include the following in your .csproj file:

```
  <PropertyGroup>
   ...
   <LangVersion>latest</LangVersion>
  </PropertyGroup>
``` 
I hope that this helps.
